### ISSUE TITLE
[android][auto] Zoom level is not synced between Android and AA

### ISSUE THREAD

**Issue by Jean-BaptisteC** [organicmaps/organicmaps#4760]
**Steps to reproduce**
1. Open OM on AA (check OM is not started on phone)
2. Use button to search location
3. Max zoom on your position
4. Open OM on phone -> map is on max zoom but map is too tiny

**Screenshots**
Max level in this version
<img src="https://user-images.githubusercontent.com/87148630/224506964-22a36e2e-1251-495d-9b38-d1d1c419be35.png" height=400 />
Max level in latest release
<img src="https://user-images.githubusercontent.com/87148630/224507002-ac802f34-af73-465a-b766-266654b03648.png" height=400 />

https://www.openstreetmap.org/#map=19/49.11609/6.17106

**System information:**
 - Operating system and its version: Android 13 + Android Auto Emulator
 - Organic Maps version: 2023.03.07-5-Google-beta
 - Device Model: Pixel 6

CC @AndrewShkrob 
Can you confirm normally phone and Android Auto is synced ? (location, position on map)

**Comment by AndrewShkrob** [comment]
Do you compare beta version from `aa` branch with release version from `master` branch?
That's incorrect.
`aa` is not always synchronized with the `master`. I think this beta apk was created before the increased zoom level.
We need to verify this bug on latest `aa` branch. I synched it with `master` today.

**Comment by Jean-BaptisteC** [comment]
I have compare beta published on firebase (OM aa) with master without aa for screenshots.

**Comment by n1snt** [comment]
Now after the latest commit, one can only use the map on either the phone or android auto (Both screens cannot view the map at the same time).

**Comment by Jean-BaptisteC** [comment]
Yes but zoom is synchronised between phone and AA, i always reproduce with latest changes
